### DISCUSSION TITLE
Fortify redirection customization

### DISCUSSION THREAD

**Discussion by CrazeTheDragon** [laravel/framework#47036] [Q&A]
Why you don`t add information about redirection customization (https://github.com/laravel/fortify/pull/298)? I lost more time when try create custom redirect after user register. I mean Why is this configuration method not described in the official documentation, although it exists and works.

**Comment by henzeb** [comment]
Simply forgotten, I guess. You can submit a PR with documentation if you wish. https://github.com/laravel/docs
